numpy/f2py/tests/test_return_character.py

```diff
@@ -82,7 +81,7 @@ class TestF77ReturnCharacter(TestReturnCharacter):
     @pytest.mark.slow
     @pytest.mark.parametrize('name', 't0,t1,t5,s0,s1,s5,ss'.split(','))
     def test_all(self, name):
-        self.check_function(getattr(self.module, name))
+        self.check_function(getattr(self.module, name), name)
```

Comment:
tried removing the `slow` mark but the return_character tests fail on s390x. Will revisit this when that is fixed.

Reply:
Wouldn't it be better to change it to a conditional xfail if that's the case, rather than hiding a failure behind a `slow` marker?

Reply:
changing